MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -525,9 +525,9 @@
 	supportedProps11.maxMultiviewViewCount = 32;
 	supportedProps11.maxMultiviewInstanceIndex = canUseInstancingForMultiview() ? uintMax / 32 : uintMax;
 	supportedProps11.protectedNoFault = false;
-	supportedProps11.maxPerSetDescriptors = 4 * (_metalFeatures.maxPerStageBufferCount +
+	supportedProps11.maxPerSetDescriptors = max(4 * (_metalFeatures.maxPerStageBufferCount +
```

Comment:
If you intent is to clamp it would be a bit more clear to use std::clamp

https://en.cppreference.com/w/cpp/algorithm/clamp

Reply:
Also minor preference ideally the clamp was a in a helper function to make the 1024 less of a magic number. Since you could comment the helper function.

Reply:
Isn't that just `max(min(value, hi), lo)`? What would I use for `hi`? I feel like that would be more verbose, because I'd have to say `std::numeric_limits<uint32_t>::max()`, just to satisfy the function's prototype.

You're probably right that I ought to factor this into a helper function--I don't know why you deleted that comment...

Reply:
Apologies you are correct std::clamp isn't suitable in this scenario.

Reply:
> I don't know why you deleted that comment...

Figures. GitHub claims to support live updates of PRs, but for some reason, after a while it just stops for me...

Oh well. Anyway, I've factored this into a helper function.